src/charts/axis.js

```diff
@@ -196,6 +198,19 @@ export default function Axis (_container) {
     }
   }
 
+  function truncateTickText (axisSelection, maxCharCount) {
+    if (!maxCharCount) {
```

Comment:
No need for change, just a comment -

Why not this here instead of the early return?

```
if (maxCharCount) {
  ...
}
```

In general, early returns make functions slightly less extensible and more error-prone.

Reply:
Maybe. I like that the logic lives at a single place, in the function instead of at each function calls (3x). There's kind of a pattern in mapd3 to call functions and leave it decide if it's time to apply it or not. I feel it brings it closer to point-free, where you just stack all functions and the logic is all encapsulated. If I could get rid of all if/else, I would :)